### PR TITLE
Review of Poetry-based Virtual Environments

### PR DESCRIPTION
**Pull request by YanlinQi-BIA**
Task Background

This PR implements updates to the Oracle puller and uploader components, focusing on managing dependencies and configurations in `poetry.lock` and `pyproject.toml`.

Order of Review

1. pyproject.toml - Updated dependencies for Oracle puller.

2. poetry.lock - Reflects dependency updates.

3. oracle_puller/ - Contains main logic updates for the Oracle puller.

4. uploader/ - Contains main logic updates for the uploader.

### CODE REVIEW

File: jobs/data_relay/oracle_puller/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "oracle_puller_daemon"
+version = "0.1.0"
+description = ""
+authors = ["Azam Beg <azam.beg@dot.ca.gov>"]
```
Comment (ian-r-rose) [inline jobs/data_relay/oracle_puller/pyproject.toml:5]: Maybe better to put @pingpingxiu-DOT-ca-gov or @YanlinQi-BIA here

File: jobs/data_relay/uploader/pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,22 @@
+[tool.poetry]
+name = "uploader"
+version = "0.1.0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```
Comment (ian-r-rose) [inline jobs/data_relay/uploader/pyproject.toml:5]: Same here

**Review by ian-r-rose — APPROVED** [review]
This looks good to me. Per our discussion yesterday on writing parquet files as an intermediate format, does it make sense to also include `pyarrow` in the oracle_puller environment?

**Comment by YanlinQi-BIA** [comment]
> > This looks good to me. Per our discussion yesterday on writing parquet files as an intermediate format, does it make sense to also include `pyarrow` in the oracle_puller environment?
> 
> Certainly, will do.

 `pyarrow` has been added to the oracle_puller environment.

**Pull request merged**